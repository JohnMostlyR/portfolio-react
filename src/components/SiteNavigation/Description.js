import styled from 'styled-components';
import mq from '../../styles/templates/mediaQueries';

const Description = styled.span`
  ${mq.m`margin-left: .5rem;`}

  @media all and (orientation: landscape) and (min-width: 37.5em) {
    display: none;
  }

  @media all and (orientation: landscape) and (min-height: 37.5em) {
    display: inline;
  }
`;

export default Description;