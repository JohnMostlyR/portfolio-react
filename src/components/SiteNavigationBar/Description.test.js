import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Description from './Description';

describe('Description', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<Description/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});