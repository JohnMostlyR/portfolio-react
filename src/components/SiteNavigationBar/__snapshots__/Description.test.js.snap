// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Should render and match the snapshot 1`] = `
@media (min-width:37.5em) {
  .c0 {
    margin-left: .5rem;
  }
}

@media all and (orientation:landscape) and (min-width:37.5em) {
  .c0 {
    display: none;
  }
}

@media all and (orientation:landscape) and (min-height:725px) {
  .c0 {
    display: inline;
  }
}

<span
  className="c0"
/>
`;
